storage/store.go

```diff
@@ -894,15 +892,6 @@ func (s *Store) BootstrapRange() error {
 	if err := engine.MVCCPutProto(batch, ms, key, now, nil, acctConfig); err != nil {
 		return err
 	}
-	// Permission config.
-	permConfig := &config.PermConfig{
-		Read:  []string{security.RootUser}, // root user
```

Comment:
this is reason `TestRangeNoGossipConfig` aren't passing now - that they were passing before was a lie due to this default config matching exactly the config in the test. The problem is a race between the test writing the new config and this method running (which publishes the configs into gossip).